CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [support-v2]
+
+### Added
+- Subscribe to `ON_VIDEO_QUALITY_ADDED`, `ON_VIDEO_QUALITY_REMOVED`, `ON_AUDIO_QUALITY_ADDED`, `ON_AUDIO_QUALITY_REMOVED` events to enable only available qualities
```

Comment:
"Support for `ON_VIDEO_QUALITY_ADDED`, `ON_VIDEO_QUALITY_REMOVED`, `ON_AUDIO_QUALITY_ADDED`, `ON_AUDIO_QUALITY_REMOVED` events in `AudioQualitySelectBox`/`VideoQualitySelectBox`"